package.json

```diff
@@ -64,14 +64,14 @@
     "eslint-plugin-node": ">=9.1.0",
     "eslint-plugin-promise": ">=4.2.1",
     "eslint-plugin-standard": ">=4.0.0",
-    "@typescript-eslint/eslint-plugin": ">=2.3.3"
+    "@typescript-eslint/eslint-plugin": ">=2.7.0"
   },
   "devDependencies": {
     "@commitlint/cli": "^8.2.0",
     "@commitlint/config-conventional": "^8.2.0",
     "@commitlint/travis-cli": "^8.2.0",
     "@types/node": "^12.12.11",
-    "@typescript-eslint/eslint-plugin": "^2.3.3",
+    "@typescript-eslint/eslint-plugin": "^2.7.0",
```

Comment:
Please go over the releases and see that I didn't miss anything that would make this release a breaking one.
https://github.com/typescript-eslint/typescript-eslint/releases